geode-dunit/src/main/java/org/apache/geode/management/internal/cli/commands/QueryCommandDUnitTestBase.java

```diff
@@ -332,31 +333,18 @@ public void testSimpleQueryWithProxyRegion() {
 
     server1.invoke(() -> prepareDataForRegion(DATA_REGION_WITH_PROXY_NAME_PATH));
 
-    String member = getHostingMember();
+    String member = "server-2";
     Random random = new Random(System.nanoTime());
     int randomInteger = random.nextInt(COUNT);
-    String query = "query --member=" + member
-        + " --query=\"select ID , status , createTime , pk, floatMinValue from "
-        + DATA_REGION_WITH_PROXY_NAME_PATH + " where ID <= " + randomInteger
-        + "\" --interactive=false";
-
-    CommandResult commandResult = gfsh.executeCommand(query);
+    StringBuilder queryString = new StringBuilder();
```

Comment:
I think there was some confusion about my original review comment. I meant to use the Geode `CommandStringBuilder` class, not the Java `StringBuilder`.